test/src/fs/promises.js

```diff
@@ -42,6 +42,13 @@ if (process.platform !== 'ios') {
       const { err } = await fs.mkdir(dirname, {})
       t.equal(err, undefined, 'mkdir does not throw')
     })
+
```

Comment:
This is not supported yet so can you remove this test to narrow the scope of just `path`